batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/controller/CustomHttpServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.lftechnology.batch7crud.controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+/**
+ * @Author Binod Shrestha <binodshrestha@lftechnology.com>
+ * Created on 1/19/16
+ */
+public abstract class CustomHttpServlet extends HttpServlet{
+    private static Logger logger = Logger.getLogger(CustomHttpServlet.class.getName());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "logger" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)